test/integration/utility.cc

```diff
@@ -110,7 +110,7 @@ IntegrationUtil::makeSingleRequest(const Network::Address::InstanceConstSharedPt
   if (!content_type.empty()) {
     headers.setContentType(content_type);
   }
-  encoder.encodeHeaders(headers, body.empty());
+  encoder.encodeHeaders(headers, body.empty()).IgnoreError();
```

Comment:
Should we fail here rather than ignore?

Reply:
Hmm yes, probably worth an ASSERT check for tests. It is impossible to happen since the headers set required headers, but worth it in case failure conditions change